R/utils_insert_db.R

```diff
@@ -230,11 +240,10 @@ insert_riskmetric_to_db <- function(pkg_name,
 insert_community_metrics_to_db <- function(pkg_name, 
                                            db_name = golem::get_golem_options('assessment_db_name')) {
   
-  if (!isTRUE(getOption("shiny.testmode")))
+  if (!isTRUE(getOption("shiny.testmode"))) {
     pkgs_cum_metrics <- generate_comm_data(pkg_name)
-  else
+ } else {
```

Comment:
Not exactly sure why the `{}` were added but they broke the community metrics. Now `pkg_cum_metrics` is only updated in shiny testmode so they are never added to the database.

Reply:
OK that's a quick fix

Reply:
fixed